packages/core/src/test/awsService/cloudWatchLogs/commands/tailLogGroup.test.ts

```diff
@@ -84,11 +90,12 @@ describe('TailLogGroup', function () {
         })
 
         // The mock stream doesn't 'close', causing tailLogGroup to not return. If we `await`, it will never resolve.
-        // Run it in the background and use waitUntil to poll its state.
+        // Run it in the background and use waitUntil to poll its state. Due to the test setup, we expect this to throw
+        // after the abortController is fired at the end of the test.
         void tailLogGroup(registry, testSource, codeLensProvider, {
             groupName: testLogGroup,
             regionName: testRegion,
-        })
+        }).catch((e) => {})
```

Comment:
So this is catching the error that was thrown to console before right? Is it worth verifying this on the catch?